packages/commands/start/index.ts

```diff
@@ -79,14 +54,41 @@ export default class Start extends Command {
         `A server (${runningInstance.name}) is already running on port ${process.env.CODECLIMBERS_SERVER_PORT} with process id ${runningInstance.pid}`,
       )
     }
+    const appUrl = `http://localhost:${process.env.CODECLIMBERS_SERVER_PORT}`
+    const WELCOME_LOGO = pc.cyan(`
+      @@@@@@@@@@@@@@@@@@@            
+      @@@@@@@@@@@@@@@@@@@            
+      @@@             @@@            
+      @@@             @@@            
+                                      
+      @@@@@@@@@@@@@@@@@@@            
+      @@@@@@@@@@@@@@@@@@@            
+      @@@             @@@            
+      @@@             @@@            
+                                      
+      @@@@@@@@@@@@@@@@@@@            
+      @@@@@@@@@@@@@@@@@@@            
+      @@@             @@@            
+      @@@             @@@            
+      `)
 
+    const WELCOME_MESSAGE = pc.white(`
+      Welcome to Code Climbers!                                                                                                                                    
+      
+      Code climbers has started and will begin tracking your activity based on the sources you add.
+      Visit ${pc.bgCyan(appUrl)} to configure your sources
```

Comment:
This looks great except I can't see the text in dark mode! Love that it auto opens
<img width="862" alt="Screenshot 2024-08-11 at 9 48 08 AM" src="https://github.com/user-attachments/assets/faa7ca6d-fa1e-403c-8e39-f926c07f1789">

Changed it to just be cyan locally and it looked better.
<img width="862" alt="Screenshot 2024-08-11 at 9 49 37 AM" src="https://github.com/user-attachments/assets/06d1e94b-e34b-4d73-8c7b-6bacded85b62">